src/Core/Program.cs

```diff
@@ -226,6 +227,15 @@ public TypeLibraryDeserializer CreateTypeLibraryDeserializer()
             return new TypeLibraryDeserializer(Platform, true, EnvironmentMetadata.Clone());
         }
 
+
+        /// The processor architectures that exist in the Program. 
+        /// <remarks>
+        /// Normally there is only one architecture. But there are examples
+        /// of binaries that have two or more processor architectures. E.g.
+        /// "fat binaries" on MacOS, or ELF binaries with both ARM32 and 
+        /// Thumb instructions.</remarks>
+        public Dictionary<string, IProcessorArchitecture> Architectures { get; }
```

Comment:
Unused property?

Reply:
This will be used shortly: users will be able to add new architectures if they want.